tests/postgres_tests/test_integration.py

```diff
@@ -0,0 +1,21 @@
+import subprocess
+import sys
+import unittest
+
+from django.test import SimpleTestCase
+
+try:
+    import psycopg2
```

Comment:
Did you consider using `PostgreSQLSimpleTestCase`? I would favor that for consistency.

Reply:
This test passes when testing against an SQLite or MySQL backend as long as psycopg2 is also installed. For this test case, I think the check `connection.vendor == 'postgresql'` skips the test too aggressively.

Reply:
I know -- there are other tests on `contrib.postgres` that could run without a database, but we ran into other requirements like `django.contrib.postgres` must be in `INSTALLED_APPS` that made it inconvenient. I think it was 36e90d1f45a13f53ce25fdc2d9c04433b835c9af.

Reply:
@timgraham yes I tried with `PostgreSQLSimpleTestCase`, but `connection.vendor` is always returning me `sqlite` locally. So such tests won't run locally.

Reply:
you can use django-box, It is very useful to run tests with other vendors. (https://github.com/django/django-box)

Reply:
@cansarigol I can do it, but it would skip for other contributors locally. There is no use for a test which gets skipped.

Reply:
> I can do it, but it would skip for other contributors locally. There is no use for a test which gets skipped.

@nasirhjafri other contributors do run tests again other backends even if they are not using django-box.

Reply:
> > I can do it, but it would skip for other contributors locally. There is no use for a test which gets skipped.
> 
> @nasirhjafri other contributors do run tests again other backends even if they are not using django-box.

Hmm thanks, I've changed it to `PostgreSQLSimpleTestCase`